extension.js

```diff
@@ -204,43 +216,30 @@ function setClipboard(text) {
     St.Clipboard.get_default().set_text(St.ClipboardType.PRIMARY, text);
 }
 
-/** creates a log menu items
- *
- * @param {Container} container is the target container
- */
-function createLogMenuItem(container) {
-    let i = new ContainerMenuItem(container.name, "logs", () => container.logs());
-    i.insert_child_at_index(createIcon("document-open-symbolic.symbolic", "action-logs"), 1);
-    return i;
-}
-
-/** creates a top menu item
- *
- * @param {Container} container is the target container
- */
-function createTopMenuItem(container) {
-    const i = new ContainerMenuItem(container.name, "top", () => container.watchTop());
-    i.insert_child_at_index(createIcon("view-reveal-symbolic.symbolic", "action-top"), 1);
-    return i;
-}
-
-/** creates a shell menu item
- *
- * @param {Container} container is the target container
- */
-function createShellMenuItem(container) {
-    const i = new ContainerMenuItem(container.name, "sh", () => container.shell());
-    i.insert_child_at_index(new St.Label({style_class: "action-sh", text: ">_"}), 1);
-    return i;
-}
-
-/** creates a stats menu item/
+/** adds a button to item and returns it
  *
- * @param {Container} container is the target container
- */
-function createStatsMenuItem(container) {
-    const i = new ContainerMenuItem(container.name, "stats", () => container.stats());
-    i.insert_child_at_index(new St.Label({style_class: "action-stats", text: "%"}), 1);
-    return i;
+ * @param item that the created button is added to
+ * @param command is the actions to executoed when clicking the button
+ * @param isconName is the icon name
+ * */
+function addButton(item, command, iconName) {
+    const btn = new St.Button({
+        reactive: true,
+        can_focus: true,
+        track_hover: true,
+        style_class: "button",
+        style: "padding-right: 10px; padding-left: 10px;",
+        x_expand: true,
+        x_align: Clutter.ActorAlign.CENTER,
+    });
+    btn.child = new St.Icon({
+        icon_name: iconName,
+        icon_size: 14,
```

Comment:
That's a very uncommon icon size (i.e. icons aren't optimized for that size).

Can you use
```suggestion
        style_class: "popup-menu-icon",
```
instead?

Reply:
Ack